### ISSUE TITLE
GitLab integration

### ISSUE THREAD

**Issue by jumski** [houndci/hound#185]
Guys what about gitlab support?
I see here and there that github api is hardcoded, do you plan/have any thoughts on this one?

There is https://github.com/NARKOZ/gitlab that can be used for interacting with gitlab.

Great job btw

**Comment by blaggacao** [comment]
@salbertson This might bubble up again. Things (adoption) has changed over the years. And especially in the enterprise GitLab is relatively widespread.
Could be an interesting market to tap.

Would you consider reopening under this circumstance?
/cc @yelizariev

**Comment by salbertson** [comment]
@blaggacao yes, I agree that things have changed a lot in the last couple years. I reopened it, let's see what input others have.

**Comment by mohitjain** [comment]
Any updates on this?

**Comment by blaggacao** [comment]
@salbertson Unfortunately, I currently cannot commit to such. :-)

**Comment by salbertson** [comment]
@blaggacao no problem, thanks anyway!

**Comment by Zegorax** [comment]
Is there any plan to implement Gitlab in the future?

**Comment by salbertson** [comment]
@Zegorax we are definitely planning on it, just hasn't been a priority. If anyone is interested in moving this along let us know.

**Comment by Zegorax** [comment]
Great !

Someone did this : https://github.com/larrylv/hound-gitlab but it's been for years ago.
Maybe you can try to use what that person did and integrate it in the main repo ?

**Comment by Zegorax** [comment]
@salbertson Is there any news about love for Gitlab ?
